hwsd/filter.cpp

```diff
@@ -135,7 +135,7 @@ class SessionFilter
 public:
     explicit SessionFilter( const std::string& name ) : name_( name ) {}
 
-    const std::string& name_;
+    const std::string name_;
```

Comment:
I remember seeing this or something very similar before, and I also remember commenting it to you. However, I can't remember whether I submitted a fix. Is this that bug? If that's not the case, then it might mean that the same bug is occurring somewhere else.